src/common/summarize-profile.js

```diff
@@ -220,6 +229,9 @@ function calculateSummaryPercentages(summary: Summary) {
       const percentage = samples / sampleCount;
       return { category, samples, percentage };
     })
+    // Sort by name first so that the results are deterministic.
+    .sort((a, b) => a.category > b.category ? -1 : 1)
+    // Sort by sample count second.
     .sort((a, b) => b.samples - a.samples);
```

Comment:
I would prefer a single sort which handles both. `a.samples == b.samples ? lexicalOrder(a.category, b.category) : b.samples - a.samples` or something like that.